src/css/custom.scss

```diff
@@ -17,6 +17,7 @@
   --ifm-color-primary-light: #00aeff;
   --ifm-color-primary-lighter: #4cc8ff;
   --ifm-color-primary-lightest: #97e1ff;
+  --ifm-color-secondary-darkest: #808080;
```

Comment:
We need something with more contrast. This doesn't pass WCAG 2 AA (4.5:1) . The contrasts are:
- 3.88 when used in the cards (#242526), 
- 4.45 when outside the cards (just below "Electron Governance")

Reply:
Digging into the Infima code (undocumented), I found some variables that work. 37afe40